integration/js/pages/AppPage.js

```diff
@@ -78,8 +78,12 @@ class AppPage {
   }
 
   async chooseUseSimulcast() {
-    let featureSelection = await this.driver.findElement(elements.optionalFeatures);
-    await featureSelection.sendKeys('simulcast');
+    let simulcastFeature = await this.driver.findElement(elements.simulcastFeature);
+    if (simulcastFeature.isSelected()) {
+      console.log('simulcast is selected');
+    } else {
+      await simulcastFeature.click();
```

Comment:
And this will raise an error because the label covers the checkbox. You should find the label with a CSS selector:

```
By.css('label[for="simulcast"]')
```

and click that instead.